ante/fee.go

```diff
@@ -53,19 +53,20 @@ func checkFees(minGasPrices sdk.DecCoins, fees sdk.Coins, gasLimit uint64) error
 		minGasPrices = sdk.DecCoins{appparams.MinMinGasPrice}
 	}
 
-	requiredFees := make(sdk.Coins, len(minGasPrices))
+	requiredFees := make(sdk.Coins, len(minGasPrices), 0)
```

Comment:
```suggestion
	requiredFees := sdk.NewCoins()
```

`make(sdk.Coins, len(minGasPrices), 0)` was causing `raw_log: "recovered: runtime error: makeslice: cap out of range\nstack:\ngoroutine`